server/start.go

```diff
@@ -149,6 +152,8 @@ which accepts a path for the resulting pprof file.
 
 	cmd.Flags().Bool(flagGRPCEnable, true, "Define if the gRPC server should be enabled")
 	cmd.Flags().String(flagGRPCAddress, config.DefaultGRPCAddress, "the gRPC server address to listen on")
+	cmd.Flags().Bool(flagGRPCWebEnable, true, "Define if the gRPC-Web server should be enabled")
```

Comment:
See my comment on the config. This needs to make note of the fact that gRPC must be enabled. Also, please be consistent with punctuation of flag descriptions.